packages/react/src/components/TopBar/TopBarMenu.test.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { OptionsMenuList } from '../OptionsMenu';
+import { fireEvent, render, screen } from '@testing-library/react';
+import TopBarMenu from './TopBarMenu';
+import { MenuBar, TopBar } from '../..';
+import { MenuItem } from '../../../lib';
+import axe from '../../axe';
+
+const [rightCode, leftCode, downCode] = [39, 37, 40];
+const [rightKey, leftKey, downKey] = ['Right', 'Left', 'Down'];
+
+const noop = () => {
+  // not empty
+};
+
+const defaultProps = {
+  id: 'foo'
+};
+
+const optionsMenu = (
+  <OptionsMenuList onClose={noop}>
+    <li>option 1</li>
+    <li>option 2</li>
+  </OptionsMenuList>
+);
+
+test('should render children', () => {
+  render(<TopBarMenu {...defaultProps}>{optionsMenu}</TopBarMenu>);
+
+  expect(
+    screen.getByRole('menuitem', { name: 'option 1' })
+  ).toBeInTheDocument();
+  expect(
+    screen.getByRole('menuitem', { name: 'option 2' })
+  ).toBeInTheDocument();
+});
+
+test('should pass-through props', () => {
+  const restProps = { foo: 'bar' };
```

Comment:
`foo` isn't a real prop we should be able to pass down if we used the `TopBarMenuProps` interface. I think we'll want to update this example.